istioctl/cmd/analyze.go

```diff
@@ -16,21 +16,30 @@ package cmd
 
 import (
 	"fmt"
+	"io"
 	"os"
 	"strings"
 
 	"github.com/spf13/cobra"
 
 	"istio.io/istio/galley/pkg/config/analysis/analyzers"
+	"istio.io/istio/galley/pkg/config/analysis/diag"
 	"istio.io/istio/galley/pkg/config/analysis/local"
 	"istio.io/istio/galley/pkg/config/meta/metadata"
 	cfgKube "istio.io/istio/galley/pkg/config/source/kube"
 	"istio.io/istio/pkg/kube"
 )
 
+type FoundAnalyzeIssuesError struct{}
+
+func (f FoundAnalyzeIssuesError) Error() string {
+	return "Found analyzer issues."
```

Comment:
Nit: "Analyzer found issues" is easier to parse and more accurate. If you change the var name to match (`AnalyzeFoundIssuesError`), this would also be consistent with the name in sysexits.go